src/userdb/meson.build

```diff
@@ -23,3 +23,14 @@ executables += [
                 'dependencies' : threads,
         },
 ]
+
+custom_target(
+        '20-systemd-userdb.conf',
+        input : '20-systemd-userdb.conf.in',
+        output : '20-systemd-userdb.conf',
+        command : [jinja2_cmdline, '@INPUT@', '@OUTPUT@'],
+        install : conf.get('ENABLE_USERDB') == 1 and sshconfdir != 'no',
+        install_dir : libexecdir / 'ssh')
+
+meson.add_install_script(sh, '-c',
+        ln_s.format(libexecdir / 'ssh' / '20-systemd-userdb.conf', sshconfdir / '20-systemd-userdb.conf'))
```

Comment:
do we need both this and the tmpfiles.d?

Reply:
the tmpfiles.d/ thing is to make sure this works on systems that come up with an empty /etc/.

The meson symlink creation is for systems that ship with an immutable but populated /etc/

Reply:
(ideally we'd have neither, and just ship the data in /usr/, but sshd doesn't support that, so we do the next best thing)